linters/polymer.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const extend = require('extend');
```

Comment:
At this point, probably not. This was a legacy thing from before Node v4. I can fix it up.